API/Async.pm

```diff
@@ -273,14 +273,17 @@ sub getFavorites {
 
 			$cb->($items);
 		},{
-			_ttl => USER_CONTENT_TTL,
+			_nocache => 1,
 			limit => MAX_LIMIT,
 		});
 	};
 
 	# use cached data unless the collection has changed or is small anyway
 	my $cached = $cache->get($cacheKey);
-	if ($cached && ref $cached->{items} && scalar @{$cached->{items}} > DEFAULT_LIMIT) {
+	if ($cached && ref $cached->{items}) {
```

Comment:
One last thing: now the comment above doesn't match any more, as you no longer run the size check.